wgengine/router/osrouter/router_windows.go

```diff
@@ -124,10 +133,6 @@ func (r *winRouter) Close() error {
 	return nil
 }
 
-func cleanUp(logf logger.Logf, interfaceName string) {
-	// Nothing to do here.
-}
```

Comment:
I like how all these dummy no-op implementations go away. Instead, they just don't set the hooks now.